apps/wallet/src/ui/app/shared/card-layout/index.tsx

```diff
@@ -1,66 +1,113 @@
 
 import type { ReactNode } from 'react';
 
-import st from './CardLayout.module.scss';
+const cardLayoutStyles = cva(
+    [
+        'flex flex-col flex-nowrap rounded-20 items-center justify-center w-popup-width h-popup-height bg-alice-blue shadow-wallet-content',
+    ],
+    {
+        variants: {
+            mode: {
+                box: 'bg-alice-blue',
+                plain: 'bg-transparent',
+            },
+        },
+        defaultVariants: {
+            mode: 'box',
+        },
+    }
+);
 
-export type CardLayoutProps = {
+export interface CardLayoutProps extends VariantProps<typeof cardLayoutStyles> {
     title?: string;
     subtitle?: string;
     headerCaption?: string;
     icon?: 'success' | 'sui';
     children: ReactNode | ReactNode[];
-    className?: string;
-    mode?: 'box' | 'plain';
     goBackOnClick?: () => void;
-};
+}
 
 export default function CardLayout({
-    className,
     children,
     title,
     subtitle,
     headerCaption,
     icon,
-    mode = 'box',
     goBackOnClick,
+    ...styleProps
 }: CardLayoutProps) {
     return (
-        <div className={cn(className, st.container, st[mode])}>
-            {goBackOnClick ? (
-                <PageTitle
-                    onClick={goBackOnClick}
-                    hideBackLabel={true}
-                    className={st.back}
-                />
-            ) : null}
-            {icon === 'success' ? (
-                <div className={st.successIcon}>
-                    <div className={st.successBg}>
+        <div className={cardLayoutStyles(styleProps)}>
+            <div className="p-7.5 pt-10 flex-grow flex flex-col flex-nowrap items-center justify-center w-full">
+                {goBackOnClick ? (
+                    <PageTitle
+                        onClick={goBackOnClick}
+                        hideBackLabel={true}
+                        className="absolute left-[22px] top-[19px]"
+                    />
+                ) : null}
+                {icon === 'success' ? (
+                    <div className="rounded-full w-12 h-12 border-dotted border-success border-2 flex items-center justify-center mb-2.5 p-1">
+                        <div className="bg-success rounded-full h-8 w-8 flex items-center justify-center">
+                            <Icon
+                                icon={SuiIcons.ThumbsUp}
+                                className="text-white text-[25px]"
+                            />
+                        </div>
+                    </div>
+                ) : null}
+                {icon === 'sui' ? (
+                    <div className="flex flex-col flex-nowrap items-center justify-center rounded-full w-16 h-16 bg-sui mb-7">
                         <Icon
-                            icon={SuiIcons.ThumbsUp}
-                            className={st.thumbsUp}
+                            icon={SuiIcons.SuiLogoIcon}
+                            className="text-white text-[34px]"
```

Comment:
`text-4xl` here as well

Reply:
Same here

Reply:
Seems like we should figure out why we have inconsistent header sizes in the design, ideally we only use sizes that are reflected in the typography overview.